velox/functions/sparksql/String.h

```diff
@@ -1019,13 +1023,17 @@ struct ConvFunction {
     // When toBase is negative, converts to signed value. Otherwise, converts to
     // unsigned value. Overflow is allowed, consistent with Spark.
     if (toBase < 0) {
+      auto isMinInt64Num =
+          unsignedValue == (uint64_t)std::numeric_limits<int64_t>::min();
```

Comment:
cc @rui-mo I change to use the unsigned number to do the comparation and pull out the check before going to `toChars` to avoid unnecessary std::abs.